pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExporter.java

```diff
@@ -270,6 +327,7 @@ public int exportDataSetAsCsv(StudyDto studyDto, List<CustomActivityExtract> act
             for (var i = 1; i <= maxInstances; i++) {
                 // Add the headers
                 headers.addAll(attributesCollector.headers());
+                headers.add(familyIdHeader);
```

Comment:
Are we sure we want to add a column for family_id for every instance? Shouldn't this be added just once? On L359 I see that we add family_id right after the participant columns, so I think this should go right after L274?